src/Common/ThreadPool_fwd.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
```

Comment:
Please add `#include <Common/ThreadPool_fwd.h>` to `ThreadPool.h` to make the compiler to check consistency of the declarations.

Reply:
Good suggestion, thanks, done.